examples/standard/version.tf

```diff
@@ -4,7 +4,7 @@ terraform {
     # Pin to the lowest provider version of the range defined in the main module to ensure lowest version still works
     ibm = {
       source  = "ibm-cloud/ibm"
-      version = "1.51.0"
+      version = ">= 1.51.0"
```

Comment:
We need to keep one example locked into the lowest version supported by the module, so maybe lets keep this at `1.51.0`

Reply:
@mounika-nalla still 1 open comment here

Reply:
This example uses the modules landing-zone-vpc (1.52.0) and key-protect-all-inclusive(1.53.0) which are set to latest versions of ibm provider. I left this example as it is now and modified the existing-cos example to lock into a specific version 1.51.0